src/App/ProductSearch/_product_search.scss

```diff
@@ -28,8 +36,11 @@
     background: $blue;
     color: #fff;
     font-size: 1em;
-    &:hover, &:focus {
+		transition: .3s;
+		&:hover, &:focus {
+			transition: .3s;
       background-color: $teal;
     }
   }
 }
+
```

Comment:
Your CSS is a million times more organized in your file than mine ever is. Nice job!